packages/explorable-explanations/src/protectedAudience/components/flowExpander.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Internal dependencies
+ */
+import app from '../app';
+import config from '../config';
+import { isInsideCircle } from '../utils';
+
+const FlowExpander = async ({ nextTipCoordinates }) => {
+  const p = app.p;
+  const iconSize = config.timeline.expandIconSize;
+  const iconRadius = iconSize / 2;
+
+  p.mouseMoved = ({ offsetX, offsetY }) => {
```

Comment:
As `app.p` is one common object, `p.mouseMoved` doesn't override the global `mouseMoved` event in p5.js?

Reply:
Right checked it and used `interestGroupCanvas or igp` object.